documentation/src/main/asciidoc/user_guide/hibernate-ogm.adoc

```diff
@@ -0,0 +1,47 @@
+==  Hibernate OGM
+Hibernate can perform CRUD operations directly on an Infinispan cluster.
+Hibernate OGM is an extension of the popular Hibernate ORM project which makes
+the Hibernate API suited to interact with NoSQL databases such as Infinispan.
+
+From the perspective of the Hibernate user, it might be interesting to replace
+the RDBMS with Infinispan to benefit from the better scalability and higher
+performance on CRUD operations; the drawback is that Infinispan - not being a
+relational database - can not run complex relational queries.
+
+From the perspective of the Infinispan user, using Hibernate OGM can simplify
```

Comment:
it can or it does? Go straight to the point.
I think this should be emphasized that to get started in minutes, OGM is a good tool (no Protobuf etc)